components/offscreen/offscreen.js

```diff
@@ -1,15 +1,20 @@
 import { css, html, LitElement } from 'lit';
 import { RtlMixin } from '../../mixins/rtl/rtl-mixin.js';
 
-export const offscreenStyles = css`
-	.d2l-offscreen {
+const offscreenStyleDeclarations = css`
+		direction: var(--d2l-document-direction, ${document.dir === 'rtl' ? css`rtl` : css`ltr`});
 		height: 1px;
 		inset-inline-start: -10000px;
 		left: -10000px;
 		overflow: hidden;
 		position: absolute !important;
 		white-space: nowrap;
 		width: 1px;
+`;
+
+export const offscreenStyles = css`
+	.d2l-offscreen {
+		${offscreenStyleDeclarations}
 	}
 	:host([dir="rtl"]) .d2l-offscreen {
```

Comment:
We can just remove this now right (and the one below)?

Reply:
Assuming `direction` only affects the logical properties, I think it's still acting as a fallback for older browsers.

Reply:
Maybe, but if anyone is using `dir` (like not from the mixin) on a component, then removing this could break stuff I guess.

Reply:
Yeah, based on the slack thread I thought we were ok with moving to "If your browser doesn't support logical properties and you're using RTL mode, you will now see the big scrollbar"

Reply:
Yeah I'm OK with making that change.